persistence/build.gradle.kts

```diff
@@ -14,10 +14,16 @@ dependencies {
 
 sqldelight {
     database("UserDatabase") {
-        dialect = "app.cash.sqldelight:sqlite-3-24-dialect:${Versions.sqlDelight}"
+        dialect = Dependencies.SqlDelight.dialect
         packageName = "com.wire.kalium.persistence"
         sourceFolders = listOf("db_user")
     }
+
+    database("KaliumDatabase") {
```

Comment:
I would suggest to use a different name instead of the project name. Maybe `GlobalDatabase` or `AppConfigDatabase` ?

Reply:
`GlobalDatabase` sounds good to me